targets/ipq40xx-generic

```diff
@@ -43,6 +43,17 @@ device('avm-fritz-repeater-1200', 'avm_fritzrepeater-1200', {
 })
 
 
+-- EnGenius
+
+device('engenius-ens620ext', 'engenius_ens620ext', {
+	factory = false,
+	extra_images = {
+		{'-squashfs-factory_30', '-factory_30', '.bin'},
+		{'-squashfs-factory_35', '-factory_35', '.bin'},
```

Comment:
Minor nit: Would it be helpful if we labeled them fw30 and fw35, to indicate that it's related to the firmware version?

Reply:
The need for multiple targets for diffrent firmware versions is thankfully not very common. So I would find it helpful to make it extra clear.

Reply:
Sure, why not. Renamed to _fw30 and _fw35.